spec/features/sign_up_spec.rb

```diff
@@ -0,0 +1,15 @@
+feature ' sign up' do
+  scenario ' a user can sign up' do
+    visit('/spaces')
+    click_button ('Sign up')
+    expect(current_path).to eq '/users/new'
+
+    fill_in('email', with: 'test@example.com')
+    fill_in('password', with: 'password123')
+    fill_in('display_name', with: 'Rubber Duck')
```

Comment:
Maybe getting ahead of myself, but it might be useful going forward to create a web_helper sign_in method to keep test code well-refactored.